assets/schemas/manifest.json

```diff
@@ -520,7 +520,7 @@
               "app.refresh",
               "app.uninstalled",
               "permit.created",
-              "permit.deleted"
+              "consent.propagated"
             ]
```

Comment:
As is apps will never get permit created events only consent

Reply:
removed it